code/go/0chain.net/smartcontract/storagesc/blobber.go

```diff
@@ -461,6 +494,51 @@ func (sc *StorageSmartContract) commitBlobberRead(t *transaction.Transaction,
 	details.ReadReward += value // stat
 	details.Spent += value      // reduce min lock demand left
 
+	rewardRound := GetCurrentRewardRound(balances.GetBlock().Round, conf.BlockReward.TriggerPeriod)
+
+	if blobber.LastRewardRoundOfDataRead >= rewardRound {
+		blobber.DataReadLastRewardRound += sizeRead
+	} else {
+		blobber.DataReadLastRewardRound = sizeRead
```

Comment:
It's a little confuse to read the DataReadLastRewardRound, not sure if it is a round number or the data size from the name. So maybe we can change it to LastRewardDataReadSize for size. And change the LastRewardRoundOfDataRead to LastRewardDataReadRound for round.

Reply:
yeah I know. I was thriving to get a better name.
LastRewardDataReadSize -  looks like the data size read in last reward, but as we're paying rewards in arrear, what we want to capture here is: what is the size read by blobber in last arrear reward round.